pootle/apps/pootle_translationproject/models.py

```diff
@@ -135,9 +133,28 @@ def for_user(self, user):
         :return: A filtered queryset with `TranslationProject`s for `user`.
         """
         if user.is_superuser:
-            return self.all()
+            return self.live()
+
+        return self.live().filter(project__disabled=False)
```

Comment:
Ah yes, in order to allow chaining, this requires to [add the methods to the QuerySet](https://docs.djangoproject.com/en/1.7/topics/db/managers/#creating-manager-with-queryset-methods). My commit exposes the same issue.